### ISSUE TITLE
Research ideas

### ISSUE THREAD

**Issue by an1lam** [adithyaparams/flipper#4]
- More extensive hyper parameter search, training data regularization like better validation set selection
  - High/low, what happens if you use a high/low train/val split to do hyperparameter selection
- How difficult would it be to generate MSAs and include a Potts baseline, checked TAPE and they didn’t include it either so maybe it’s not low hanging fruit
- Testing new ESM models + [Ankh](https://github.com/agemagician/Ankh) LM
- Does fine-tuning one final layer of the ESM model(s) improve performance?
